pkgs/development/python-modules/pytest-xdist/setup-hook.sh

```diff
@@ -1,14 +1,13 @@
 pytestXdistHook() {
     pytestFlagsArray+=(
         "--numprocesses=$NIX_BUILD_CORES"
+        "--forked"
     )
 
-    # Calling os.fork on darwin leads to crashes
-    # https://github.com/python/cpython/issues/77906
+    # Using --forked on darwin leads to crashes when fork safety is
+    # enabled. https://github.com/python/cpython/issues/77906
     if [[ "$OSTYPE" != "darwin"* ]]; then
```

Comment:
I think this condition needs to be inverted now.

Reply:
Of course it does, thank you.